adapter/exporter/src/main/java/org/mobilitydata/gtfsvalidator/exporter/JsonNoticeExporter.java

```diff
@@ -226,4 +226,4 @@ public void export(RouteLongNameEqualsShortNameNotice toExport) throws IOExcepti
     public void export(RouteLongNameContainsShortNameNotice toExport) throws IOException {
         jsonGenerator.writeObject(toExport);
     }
-}
+}
```

Comment:
It is best practice to have newline characters at the end of files

Reply:
Opening an issue on this matter since various files are affected